README.md

```diff
@@ -7,6 +7,12 @@ Method Cards
 - To build a shared vocabulary for each method among 18F staff and our partner agencies.
 - To give less experienced or new researchers a gateway into well-documented and proven research methods, supported by other 18F team members.
 
+## Method Cards mission and vision
+In late 2016, we articulated a vision and mission for the project that focus our efforts on meeting the needs of government users:
```

Comment:
I think the methods apply outside the world of government as well. I would recommend removing "government" from the mission/vision. Thoughts?

Reply:
While we know folks outside gov use the cards, the team and product owner at the time (through various mission/visions setting activities) made the intentional choice to narrow our primary focus just a bit and target gov users and what we could offer specific to folks working in gov, with the idea that this would not exclude folks outside the gov who want to use the cards. Happy to revisit though!

Reply:
@andrewmaier love to get your thoughts on this :)

Reply:
This is a great start! I like to think of vision as a way of completing the phrase "We envision a world where..." 

So I might say our vision involves something like "increasing the degree to which teams collaboratively solicit and incorporate additional perspectives into their design process." 

That vision, to my mind, would lead to a mission centered around demystifying human-centered design, and lowering the barriers to entry to an ongoing HCD practice (things like providing guidance for choosing, adopting, sequencing, and adapting these tools to government).